src/compat_win32.cpp

```diff
@@ -7,7 +7,7 @@
  */
 #if defined (_WIN32) && !defined (__CYGWIN__)
 
-#include "windows_compat.h"
+#include "compat.h"
```

Comment:
What is the reason for this change? 
Note:
* In your previous commits you had a `compat_win32.h` file which apparently wasn't used.
* Your new `src/uncrustify.h` file is including `windows_compat.h`